src/main/webapp/download.html

```diff
@@ -68,7 +68,7 @@ <h3 class="mb-0 text-dark">Karaf Container <span class="text-muted">4.2.0</span>
               <div class="mb-1 text-muted">April 9, 2018</div>
               <p class="card-text mb-auto">
                 Binary Distribution :
-                  <a href="http://www.apache.org/dyn/closer.lua/karaf/4.2.0/apache-karaf-4.2.0.tar.gz">tar.gz</a> [<a href="https://www.apache.org/dist/karaf/4.2.0/apache-karaf-4.2.0.tar.gz.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/4.2.0/apache-karaf-4.2.0.tar.gz.sha512">SHA512</a>] - 
+                  <a href="http://www.apache.org/dyn/closer.lua/karaf/4.2.0/apache-karaf-4.2.0.tar.gz">tar.gz</a> [<a href="https://www.apache.org/dist/karaf/4.2.0/apache-karaf-4.2.0.tar.gz.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/4.2.0/apache-karaf-4.2.0-src.tar.gz.sha512">SHA512</a>] - 
```

Comment:
It's not `src` here but binary. I don't think it has to be changed.

Reply:
Right, I'm fixing that